addons/native_dialog/gtk_filesel.c

```diff
@@ -46,6 +46,10 @@ static gboolean create_gtk_file_dialog(gpointer data)
 
    _al_gtk_make_transient(display, window);
 
```

Comment:
The convention is first and foremost to stick to the convention used by the file you are in, since unfortunately we're not completely consistent. We do generally tend to use 3 spaces for indents and use {} around if blocks, amongst others.